src/main/java/org/zalando/fahrschein/domain/Cursor.java

```diff
@@ -1,6 +1,6 @@
 package org.zalando.fahrschein.domain;
 
-public final class Cursor {
+public class Cursor {
```

Comment:
There should be alway a possibility to mock away dependencies imho. Otherwise provide a interface.